certification/internal/engine/engine.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
 	containerutils "github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/utils/container"
-	fileutils "github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/utils/file"
+	fileutils "github.com/redhat-openshift-ecosystem/openshift-preflight/certification/utils"
```

Comment:
To avoid confusion this should be 'certutils'

Reply:
Done!